### PR TITLE
Extractor team

### PR DESCRIPTION
**Pull request by neil-nb**
Redid the main menu screen, among other things like re-refining ship sprite. (Most of the things I did, I did in other people's branches so this is not a good representation of the work I did)

### CODE REVIEW

File: source/core/assets/configs/player.json
Comment (zombiechook) [inline source/core/assets/configs/player.json:1]: This file probably shouldn't have been added to the commit

**Review by zombiechook — DISMISSED** [review]
The only issue is the change to player.json. Revert the health back to 100 and this should be good to merge

**Review by rowan-gray — APPROVED** [review]
Menu looks much better

**Review by kk4w4i — APPROVED** [review]
lgtm

**Review by samv26 — APPROVED** [review]
Nice

**Review by samv26 — APPROVED** [review]
good

**Review by samv26 — APPROVED** [review]
Nice

**Review by samv26 — APPROVED** [review]
G

**Pull request merged**